sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -102,11 +105,13 @@
     public ExpressionAnalyzer(Functions functions,
                               SessionContext sessionContext,
                               com.google.common.base.Function<ParameterExpression, Symbol> convertParamFunction,
-                              FieldProvider fieldProvider) {
+                              FieldProvider fieldProvider,
+                              @Nullable SubqueryAnalyzer subQueryAnalyzer) {
```

Comment:
adding another ctor without this arg instead of calling it lot of times with `null` makes sense maybe ;)